mysql-test/suite/rocksdb/t/drop_table-master.opt

```diff
@@ -1,4 +1,3 @@
 --rocksdb_max_subcompactions=1
---rocksdb_info_log_level=info_level
```

Comment:
@spetrunia actually the reason we have this in here right now is because we have noticed the rocksdb.drop_table test failing intermittently due tripping an assertion with manual compaction in rocksdb.   It was added in https://github.com/facebook/mysql-5.6/commit/977988340eff5cfbf651c5ccdb80c857c4c26a6f.  

This issue still isn't resolved so I don't think we should remove this just yet.